server/app.js

```diff
@@ -1,9 +1,16 @@
 const express = require('express')
 const path = require('path')
 const app = express();
-const { route } = require('./server')
-app.use('/api/websites', route);
-
+const { searchRoute } = require('./search');
+const { homeRoute } = require('./home');
+const { profileRoute } = require('./profile');
+const { loginRoute } = require('./login');
+const { reviewRoute } = require('./review');
+app.use('/api/websites', searchRoute);
+app.use('/home', homeRoute);
```

Comment:
```suggestion
app.use('/', homeRoute);
```
Everything here works great, just a recommendation of changing the "home" to a blank back-slash so that the homepage will be displayed with just the default url.